html/pfappserver/root/authentication/source/read.tt

```diff
@@ -21,7 +21,7 @@
         [% form.field('id').render | none %]
         [% form.field('description').render | none %]
         [% INCLUDE "authentication/source/type/${source.type}.tt" %]
-        [% IF source.type != 'Chained' %]
+        [% IF source.type != 'Chained' && source.type != 'HTTP' %]
```

Comment:
Maybe we should have a source attribute or method ignore rules? Because I have similar code in billing as a source